webui/master/src/containers/pages/Metrics/Metrics.tsx

```diff
@@ -290,6 +291,33 @@ export class MetricsPresenter extends React.Component<AllProps> {
             </Table>
           </div>
         ))}
+        <div className="col-12">
+          <h5>Alluxio Master Journal Disk Status</h5>
+          <Table hover={true}>
+            <tbody>
+              <tr>
+                <th>Device Path</th>
+                <th>Allocated Bytes</th>
+                <th>Used Bytes</th>
+                <th>Available Bytes</th>
+                <th>Percent Available</th>
+                <th>Mount Path</th>
+              </tr>
+              {data.journalDiskMetrics.map((info: IJournalDiskInfo, idx: number) => {
+                return (
+                  <tr key={idx}>
+                    <td>{info.diskPath}</td>
+                    <td>{info.totalAllocatedBytes}</td>
+                    <td>{info.usedBytes}</td>
+                    <td>{info.availableBytes}</td>
```

Comment:
(nit) It would be nice to have a more human readable format (auto converting to MB,GB) when displaying the numbers. Maybe a follow up PR?

Reply:
updated in latest commit:
![image](https://user-images.githubusercontent.com/3464508/115133992-5a742e00-9fc1-11eb-8eb8-7d7d7318118c.png)

@tieujason330 FYI since I changed some UI code